app/Http/Controllers/AdminDash.php

```diff
@@ -1276,6 +1291,48 @@ public function removeLocalHeroWinner($id, $year, $month) {
         return redirect('/dashboard/admin/bronze-mic/localsort/'.$year.'/'.$month)->with('success', 'The local hero winner has been removed successfully.');
     }
 
+    public function updateLocalHeroChallenge(Request $request, $id) {
+        $validator = $request->validate([
+            'title' => 'required',
+            'description' => 'required'
+        ]);
+
+        $local_hero_challenge = LocalHeroChallenges::find($id);
+        if (!$local_hero_challenge) {
+            $local_hero_challenge = new LocalHeroChallenges;
+            $local_hero_challenge->year = $request->year;
+            $local_hero_challenge->month = $request->month;
+            $news = new Calendar;
+        } else {
+            $news = Calendar::find($local_hero_challenge->news_id);
+            if (!$news) {
+                $news = new Calendar;
+            }
+        }
+        $news_pre_title = Carbon::create()->day(1)->month($request->month)->format('F') . " Challenge: ";
+        $news->title = $news_pre_title . $request->title;
+        $news->date = Carbon::now()->format('m/d/Y');
+        $news->body = $request->description;
+        $news->created_by = Auth::id();
+        $news->type = 2;
+        $news->visible = ($request->postToNews == 1) ? 1 : 0;
+        $news->save();
+
+        $local_hero_challenge->positions = is_array($request->positions) ? implode(', ', $request->positions) : '';
```

Comment:
Should we store this without a space? I.e. `implode(',', ...)...`?

Reply:
Agreed and fixed